REFERENCE.md

```diff
@@ -267,7 +278,7 @@ If you observe poor performance on a dataset with a small number of examples, th
 
 ### The variance estimates are jumpy or very large.
 
-In this case, it would be good to try growing a larger number of trees. Obtaining good variance estimates often requires growing more trees than it takes to only obtain accurate predictions.
+In this case, it would be good to try growing a larger number of trees. Obtaining good variance estimates often requires growing more trees than it takes to only obtain accurate predictions. See the discussion under [`num.trees`](#num-trees) and [Merging Forests](#merging-forests).
```

Comment:
The hyperlink to `num.trees` didn't work for me when I tried it.

Reply:
Sorry! -- fixed.